mirdata/annotations.py

```diff
@@ -55,7 +55,12 @@
 }
 
 #: Section units
-SECTION_UNITS = {"open": "no scrict schema or units"}
+SECTION_UNITS = {
```

Comment:
We would like to avoid dataset-specific units, so we suggest changing the name and the description to a more general ones. We propose the following although we are open to other options so feel free to propose them.

_"salami_function": "Segment annotations with functional labels from the SALAMI guidelines"_ --> `functional: Segment annotations with functional labels. E.g. intro, verse, chorus.`
_"salami_upper": "Segment annotations with SALAMI’s upper-case (large) label format"_ --> `alphabetical_upper: Segment annotations with alphabetical upper-case (high-level) format. E.g. ABCA`
_"salami_lower": "Segment annotations with SALAMI’s lower-case (small) label format"_  --> `alphabetical_lower: Segment annotations with alphabetical lower-case (low-level) format. E.g. aa’bb’ ccd bb’ aa’bb’ (ABCA)`